web/packages/teleport/src/Discover/ConnectMyComputer/SetupConnect/SetupConnect.tsx

```diff
@@ -320,29 +323,6 @@ export const usePollForConnectMyComputerNode = (args: {
   return { node, isPolling };
 };
 
-type DownloadLink = { text: string; url: string };
-
-const DownloadConnect = (props: { downloadLinks: Array<DownloadLink> }) => {
-  if (props.downloadLinks.length === 1) {
-    const downloadLink = props.downloadLinks[0];
-    return (
-      <ButtonSecondary as="a" href={downloadLink.url}>
-        Download Teleport Connect
-      </ButtonSecondary>
-    );
-  }
-
-  return (
-    <MenuButton buttonText="Download Teleport Connect">
-      {props.downloadLinks.map(link => (
-        <MenuItem key={link.url} as="a" href={link.url}>
-          {link.text}
-        </MenuItem>
-      ))}
-    </MenuButton>
-  );
-};
-
 function getConnectDownloadLinks(
```

Comment:
Would it make sense to extract `getConnectDownloadLinks` into `DownloadConnect.tsx` as well? I imagine the logic to get a link will be pretty much the same, no?

Reply:
yes, good call. moved into the file as well